src/domain/entities/Instance.ts

```diff
@@ -0,0 +1,21 @@
+export interface InstanceData {
```

Comment:
Forgot about this. Typically I don't think we want instances as entities, as it's in the data domain.

Reply:
True, in some apps it's domain but normally it's data, will do